.github/workflows/pull-request.yml

```
@@ -23,3 +23,20 @@ jobs:
 
       - name: run yamllint
         run: yamllint .github/workflows/*.yml
+  checkpatch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v3
```
v2 has started to print warnings, see for instance: 
https://github.com/thesofproject/sof/pull/6720

See, some things did change :-)